### PR TITLE
chore(starknet_consensus_manager): add chain_id to config

### PR DESCRIPTION
**Pull request by Yael-Starkware**

### CODE REVIEW

**Comment by Yael-Starkware** [comment]
* **#2433** <a href="https://app.graphite.dev/github/pr/starkware-libs/sequencer/2433?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/starkware-libs/sequencer/2433?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: crates/sequencing/papyrus_consensus/src/config.rs [hunk 1]
```diff
@@ -45,6 +48,12 @@ pub struct ConsensusConfig {
 impl SerializeConfig for ConsensusConfig {
     fn dump(&self) -> BTreeMap<ParamPath, SerializedParam> {
         let mut config = BTreeMap::from_iter([
+            ser_param(
+                "chain_id",
+                &self.chain_id,
+                "The chain id of the Starknet chain.",
```
Comment (guy-starkware) [inline crates/sequencing/papyrus_consensus/src/config.rs:54]: Could be nice to have a couple of examples like mainnet/sepolia. (I don't know if this is something we usually do in config descriptions).

**Review by guy-starkware — COMMENTED** [review]
Looks good. I have one suggestion...

File: crates/sequencing/papyrus_consensus/src/config.rs (hunk 1 above)
Comment (Yael-Starkware) [reply crates/sequencing/papyrus_consensus/src/config.rs:54]: I added documentation for details.

**Review by Yael-Starkware — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2433)* status: 5 of 6 files reviewed, 1 unresolved discussion (waiting on @alonh5 and @guy-starkware)

**Review by yair-starkware — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2433)* status: all files reviewed, 1 unresolved discussion (waiting on @alonh5 and @guy-starkware)

**Review by yair-starkware — APPROVED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2433)* status: all files reviewed, 1 unresolved discussion (waiting on @alonh5 and @guy-starkware)

**Review by guy-starkware — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2433#-:-ODL281hAJTukog6itSM:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @alonh5)

**Pull request merged**